egui/src/style.rs

```diff
@@ -145,6 +149,30 @@ pub struct Style {
     /// The [`FontFamily`] and size you want to use for a specific [`TextStyle`].
     ///
     /// The most convenient way to look something up in this is to use [`TextStyle::resolve`].
+    ///
+    /// If you would like to overwrite app text_styles
+    ///
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// # let mut ctx = egui::Context::default();
```

otherwise the doctest will fail